manifests/golang.yml

```diff
@@ -527,7 +527,7 @@ tests/:
   test_library_conf.py:
     Test_HeaderTags: v1.53.0
     Test_HeaderTags_Colon_Leading: v1.53.0
-    Test_HeaderTags_Colon_Trailing: bug (AIT-8571)
+    Test_HeaderTags_Colon_Trailing: v1.70.0 # v1.69.0?
```

Comment:
@cbeauchesne , let me know which you prefer.

Reply:
Is it working on the last released version (which is `1.69.0`), then the declaration must be `v1.69.0`.

----

Or is it working on your main branch? then it needs to be activated on it. You can't use `1.70.0` as this version does not exist yet.

The version declared for your main branch is `1.70.0-dev` : https://github.com/DataDog/system-tests/actions/runs/11386647644/job/31679548008?pr=3257#step:13:22

In semver `1.70.0-dev` < `1.70.0`. So you need to use `v1.70.0-dev`.